README.md

```diff
@@ -110,8 +110,14 @@ so it runs anywhere Git and Bash runs.
 * __Autocompletion for [Fish shell](http://fishshell.com)__
 
     ```
-    mkdir -p ~/.config/fish/completions/
-    curl -o ~/.config/fish/completions/git.fish http://raw.githubusercontent.com/Originate/git-town/master/autocomplete/git.fish
+    $ git town install-fish-autocompletion
+    ```
+
+    or manually
+
+    ```
+    $ mkdir -p ~/.config/fish/completions/
+    $ curl -o ~/.config/fish/completions/git.fish http://raw.githubusercontent.com/Originate/git-town/master/autocomplete/git.fish
```

Comment:
I don't see any reason to list the manual steps.

I also think this entire section of the README can be removed and just update the summary of git-town to be 
```
git-town - general Git Town help, view and change Git Town configuration, install autocompletion
```